gdc_ng_models/models/release_log.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Boolean, Column, Enum, Index, Text
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+
+class ReleaseLog(Base):
+
+    __tablename__ = "release_log"
+    __table_args__ = (
+        Index("release_log_project_id_idx", "program_name", "project_code"),
+    )
+
+    def __repr__(self):
+        return "<ReleaseLog(project_id={}-{}, release_number='{}', open='{}')>".format(
+            self.program_name,
+            self.project_code,
+            self.release_number,
+            self.open,
+        )
+
+    program_name = Column(Text, primary_key=True, nullable=False)
```

Comment:
I think it's possible to have multiple records with the same `(program_name, project_code)` pair, so I think you still need an autoincrement `id` field with `id_seq` defined.

Reply:
Oh! Good catch. I mixed the two tables up and thought this one was the one that would be modified.

Reply:
To expand on this:
Only `id` should be the primary key.  If we want to add a uniqueness, we should use a unqiue `Index`.

Reply:
The ticket had release_number as part of the composite of keys, which I think should make it unique. :thinking:

Reply:
We could potentially release multiple data types and both open- and controlled-access data in a single release, so we would either need to make all five columns here part of the primary key or use a surrogate key. I don't have a strong preference, but I think the surrogate key makes more sense. Phil's comment still applies in that case, though.